### PR TITLE
Add support for custom G.A.M.M.A repository

### PR DESCRIPTION
**Pull request by sobkas**
Ability to use custom G.A.M.M.A repository from Github
I don't think there are other sites with forks so for now only Github support

### CODE REVIEW

**Comment by sobkas** [comment]
Any ideas?

File: launcher/commands/install.py [hunk 1]
```diff
@@ -181,6 +181,12 @@ def _create_full_install_args() -> Dict:
             "dest": "custom_def",
             "default": None,
         },
+        "--custom-gamma-repository": {
+            "help": "Set a custom repository for S.T.A.L.K..E.R.: G.A.M.M.A.",
+            "type": str,
+            "dest": "custom_repo",
+            "default": None,
```
Comment (Mord3rca) [inline launcher/commands/install.py:188]: `"default": "Grokitach/Stalker_GAMMA",`

File: launcher/commands/install.py [hunk 2]
```diff
@@ -215,6 +221,7 @@ def __init__(self):
         self._dl_dir = None
         self._mod_dir = None
         self._grok_mod_dir = None
+        self._repo = 'Grokitach/Stalker_GAMMA'
```
Comment (Mord3rca) [inline launcher/commands/install.py:224]: `self._repo = None`

File: launcher/commands/install.py [hunk 3]
```diff
@@ -322,6 +329,8 @@ def run(self, args):
             GammaSetup().run(args)
 
         # Start installing
+        if args.custom_repo:
+            self._repo = args.custom_repo
```
Comment (Mord3rca) [inline launcher/commands/install.py:333]: Always set repo from args since now, default value is correct

**Review by Mord3rca — CHANGES_REQUESTED** [review]

**Comment by Mord3rca** [comment]
Sorry, a bit busy at the moment.

Only Github support for now is fine. Minor changes to do and I'll find a moment to test this

**Comment by Mord3rca** [comment]
Merged in #150

**Pull request closed without merging**